wpiutil/src/main/native/include/wpi/StringMap.h

```diff
@@ -21,7 +21,7 @@
 #include "wpi/MemAlloc.h"
 #include "wpi/PointerLikeTypeTraits.h"
 #include "wpi/ErrorHandling.h"
-#include "wpi/deprecated.h"
+#include "wpi/deprecated"
```

Comment:
Should be deprecated.h, and there was another extra removal below.

Reply:
Ah, sorry about that one.

I didn't get the meaning of "there was another extra removal below" - the rest of the commit was lines re-added to files?